persistent/Database/Persist/Quasi.hs

```diff
@@ -413,11 +413,14 @@ Unfortunately, we can't use this to create Haddocks for you, because <https://gi
 -}
 module Database.Persist.Quasi
     ( parse
-    , PersistSettings (..)
+    , PersistSettings
```

Comment:
I noticed that the fields for `PersistSettings` were not actually being hidden, though from your comment I assume this is the desired behaviour in this version of persistent going forward, so I have snipped the fields off the export here.

Reply:
oh, dang, good catch 👍🏻